README.md

```diff
@@ -1,2 +1,184 @@
 # trampss-redux-data-store
+
+Factory of Redux reducers and their associated actions and selectors.
+> Make your Redux code base tinier and simpler to maintain
+
 [![CircleCI](https://circleci.com/gh/Trampss/trampss-redux-data-store.svg?style=shield)](https://circleci.com/gh/Trampss/trampss-redux-data-store) [![Coverage Status](https://coveralls.io/repos/github/Trampss/trampss-redux-data-store/badge.svg?branch=master)](https://coveralls.io/github/Trampss/trampss-redux-data-store?branch=master) [![NPM Version](https://badge.fury.io/js/trampss-redux-data-store.svg)](https://www.npmjs.com/package/trampss-redux-data-store)
+
+## Contents
+ - [Purpose](#purpose)
+ - [Why](#why)
```

Comment:
I don't understand. If you're asking me the purpose of the `why` section, this is an advice given by @jlandure and @frinyvonnick about how to structure a README.md :)